app/components/common/TeamLeaderboardTable.component.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Image, Table, List} from 'semantic-ui-react';
+import { FormattedNumber } from 'react-intl';
+import './style.scss';
+
+const TeamLeaderboardTable = ({
+  data, prefix, image, dataKey, className, height, teams
+}) => (
+  <div>
+    <List style={{ height: height }} className={className}>
+      <Table celled collapsing basic='very'>
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell>#</Table.HeaderCell>
+            <Table.HeaderCell>Team Name</Table.HeaderCell>
+            <Table.HeaderCell>Distance</Table.HeaderCell>
+            <Table.HeaderCell>Finish Date</Table.HeaderCell>
+          </Table.Row>
+        </Table.Header>
+        <Table.Body>
+          {teams.map((item, index) => (
+            <Table.Row key={item.name}>
+              <Table.Cell className="position">{index + 1}.</Table.Cell>
+              <Table.Cell className="team">
+                {/* {image ? <Image avatar src={require(`images/${item.name}.png`)} />
```

Comment:
I'll add an svg avatar so I think this can be removed

Reply:
Yep, removed after removing image prop!